contracts/temporary/FlowFreeze.cdc

```diff
@@ -0,0 +1,47 @@
+/*
+
+    FlowFreeze
+
+    This contract gives an admin the ability to freeze/unfreeze accounts
+    by adding them to a list stored in the contract.
+
+    If the account is on the frozen list, it will not be able to send transactions
+    and any transaction that accesses its account state will fail.
+
+    The execution envorionment will automatically query this contract as part of every transaction.
+
+    This is a TEMPORARY CONTRACT that will be in place until the proper protections
+    are in place with fees and the network is more decentralized
+
+ */
+
+
+pub contract FlowFreeze {
+
+    access(account) var freezeList: {Address: Bool}
+
+    pub let AdminStoragePath: StoragePath
+
+    pub resource Admin {
+
+        pub fun freezeAccount(_ address: Address) {
+            FlowFreeze.freezeList[address] = true
```

Comment:
do we need to add: https://github.com/onflow/flow-go/pull/329/files#diff-e93a1f3993224e399ffe091fd8e6537b5cfcf0350249ea2a621096d4d01b4d59R61

```
setAccountFrozen(0x%s, true)
```